code/game/gamemodes/blob/theblob.dm

```diff
@@ -185,6 +190,64 @@
 	update_icon()
 	return
 
+/obj/effect/blob/attack_hand(mob/living/carbon/human/H as mob)
+	if(!H.is_hand_valid_for_attack()) return
+	if(H.gloves && istype(H.gloves,/obj/item/clothing/gloves))

```

Comment:
The human attack hand needs to be refactored for symmetry then

I suppose it may as well be in this pr